docs/ci-services.md

```diff
@@ -0,0 +1,102 @@
+# CI Services
+
+The CI services map defines the supported CI services the user will be
+presented with when running the scaffolder.
+
+## Example
+
+```javascript
+{
+  CIServiceName: {scaffolder: opts => scaffold(opts), public: true}
+}
+```
+
+## Existing scaffolders
+There are already multiple CI service scaffolders [available for
+consumption](https://github.com/form8ion/awesome#continuous-integration-services).
+Should your workflow fit into those, feel free to integrate them directly. If
+not, you are welcome and encouraged to build more (and maybe consider
+submitting a PR with a link 😎)!
+
+## Config
+
+### keys: __string__
+
+Keys in the CI services map define the choices the user will be presented with
+when choosing their desired CI service. These have no functional impact on the
+scaffolder.
+
+example: `CIServiceName`
+
+### values: __object__
+
+Values in the CI services map define the actual mechanisms and configuration
```

Comment:
```suggestion
Values in the CI services map define the actual plugins and options
```

does this fit your goal with this piece? if so, they are maybe a little more consistent with the terminology ive been using